artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConsumerThread.java

```diff
@@ -81,7 +81,7 @@ public void browse() {
          while (enumBrowse.hasMoreElements()) {
             Message msg = enumBrowse.nextElement();
             if (msg != null) {
-               System.out.println(threadName + " Received " + (msg instanceof TextMessage ? ((TextMessage) msg).getText() : msg.getJMSMessageID()));
+               System.out.println(threadName + " Browsing " + (msg instanceof TextMessage ? ((TextMessage) msg).getText() : msg.getJMSMessageID()));
```

Comment:
consistency of casing, maybe lowercase Browsing? either that or make others Capitalised.